build/ansible/roles/supervisord/files/grafana.ini

```diff
@@ -10,7 +10,7 @@ command =
 user = pmm
 directory = /usr/share/grafana
 autorestart = true
-autostart = true
+autostart = false
```

Comment:
if we don't autostart it, do we really need this file?

Reply:
Good catch 👍 
I have not tested, but maybe we don't need it.